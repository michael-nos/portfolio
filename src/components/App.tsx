export const App = () => (
  <div className='app'>
    <div className='app-corner-triangle'></div>
    <div className='app-landing-content'>
      <div className='app-large-heading-text'>Hey,</div>
      <div className='app-large-heading-text'>I'm Michael,</div>
      <div className='app-large-heading-text'>software engineer</div>
      <div className='app-heading-sub-text'>React / TypeScript / GraphQL</div>
      <div className='app-bio-container'>
        <p>
          I'm a software engineer with experience in software development using
          both monolithic and microservice architecture. I enjoy learning about
          new technologies to further develop my skillset and to embrace the
          change in the ever-expanding web ecosystem.
        </p>
      </div>
      <div className='app-connect-text-container'>
        <p>
          Want to connect? Feel free to reach out on{' '}
          <a
            className='linkedin-link-text'
            href='https://www.linkedin.com/in/michael-nos/'
            target='_blank'
          >
            LinkedIn
          </a>
          .
        </p>
      </div>
    </div>
  </div>
);
