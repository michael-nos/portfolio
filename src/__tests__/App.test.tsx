import { App } from "@/components/App";
import { render, screen } from "@testing-library/react";

describe("App", () => {
	it("should match the snapshot", () => {
		const { container } = render(<App />);
		expect(container).toMatchSnapshot();
	});

	it("should render the heading text", () => {
		render(<App />);
		expect(screen.getByText(/I'm Michael/i)).toBeInTheDocument();
	});
});
