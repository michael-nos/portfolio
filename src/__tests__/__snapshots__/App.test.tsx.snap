// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > should match the snapshot 1`] = `
<div>
  <div
    class="app"
  >
    <div
      class="app-corner-triangle"
    />
    <div
      class="app-landing-content"
    >
      <div
        class="app-large-heading-text"
      >
        Hey,
      </div>
      <div
        class="app-large-heading-text"
      >
        I'm Michael,
      </div>
      <div
        class="app-large-heading-text"
      >
        software engineer
      </div>
      <div
        class="app-heading-sub-text"
      >
        React / TypeScript / GraphQL
      </div>
      <div
        class="app-bio-container"
      >
        <p>
          I'm a software engineer with experience in enterprise software development using both monolithic and microservice architecture. I enjoy learning about new technologies to further develop my skillset and to embrace the change in the ever-expanding web ecosystem.
        </p>
      </div>
      <div
        class="app-connect-text-container"
      >
        <p>
          Want to connect? Feel free to reach out on
           
          <a
            class="linkedin-link-text"
            href="https://www.linkedin.com/in/michael-nos/"
            target="_blank"
          >
            LinkedIn
          </a>
          .
        </p>
      </div>
    </div>
  </div>
</div>
`;
